multipaz/src/androidInstrumentedTest/kotlin/org/multipaz/testUtil.android.kt

```diff
@@ -0,0 +1,13 @@
+package org.multipaz
+
+import org.bouncycastle.jce.provider.BouncyCastleProvider
+import java.security.Security
+
+actual fun testUtilSetupCryptoProvider() {
+    // On Android (in tests), load BouncyCastleProvider so we can exercise
+    // all the tests involving Brainpool curves.
+    // We use addProvider() instead of insertProviderAt() to avoid making BC the
+    // default provider, which can cause ClassCastExceptions with standard java interfaces.
+    Security.removeProvider(BouncyCastleProvider.PROVIDER_NAME) // remove first to ensure it's not already there
+    Security.addProvider(BouncyCastleProvider()) 
+}
```

Comment:
Add same printing as in testUtilSetupCryptoProvider() implementations for other platforms:
```
    println("In testUtilCommonSetup for androidInstrumentedTest")
    println("Crypto.provider: ${Crypto.provider}")
    println("Crypto.supportedCurves: ${Crypto.supportedCurves}")
```